pkg/webhook/v1beta1/webhook.go

```diff
@@ -21,27 +21,29 @@ import (
 
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 
 	"github.com/kubeflow/katib/pkg/webhook/v1beta1/experiment"
 	"github.com/kubeflow/katib/pkg/webhook/v1beta1/pod"
 )
 
 func AddToManager(mgr manager.Manager, port int) error {
 	// Create a webhook server.
-	hookServer := &webhook.Server{
```

Comment:
The `webhook.Server` was replaced with interface one:

https://github.com/kubernetes-sigs/controller-runtime/blob/7f0c6dc440f334849b25866dd9990ac1a5a777c2/pkg/webhook/server.go#L49-L68